wicket-core/src/test/java/org/apache/wicket/markup/renderStrategy/ChildFirstHeaderRenderStrategyTest.java

```diff
@@ -51,6 +57,39 @@ public void test2() throws Exception
 	}
 
 	/**
+	 * Tests that when a controller of an enclosure is added to the ajax target, its header
+	 * contributions reach the response
+	 *
+	 * WICKET-6459
+	 *
+	 */
+	@Test
+	public void testAjaxAndEnclosures() throws Exception
+	{
+
+		tester.startPage(SimplePage3.class);

```

Comment:
Let's rename the page name to `InlineEnclosureAjaxRenderPage`.
`SimplePage`s are legacy, but for new stuff we should try better! :-)

Reply:
I renamed it to `EnclosureAjaxRenderPage`, since it actually tests for both regular and inline enclosures